sdk/remoterendering/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -136,9 +141,13 @@ def begin_asset_conversion(self, conversion_id, input_settings, output_settings,
             **kwargs)
 
         polling_method = ConversionPolling(account_id=self._account_id, polling_interval=polling_interval)
+        def deserialization_method(input):
+            # type: (Any) -> AssetConversion
+            raise Exception("Not implemented")
+
         return LROPoller(client=self._client,
                          initial_response=initial_state,
-                         deserialization_callback=lambda: None,
+                         deserialization_callback=deserialization_method,
```

Comment:
Am I missing something? It seems like we used to return None on the callback, but now we raise an exception?